packages/sdks/python/tester-local/pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Your Name <you@example.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = "~3.11"
```

Comment:
Tie to python version 3.11 is there are incompatibilities with anything below and above 3.11